_includes/body-landing.html

```diff
@@ -0,0 +1,844 @@
+                  Troubleshoot volumes or bind mount errors
+                </a>
+              </p>
+              <p>
+                <a href="https://success.docker.com/article/troubleshooting-container-networking">
+                  Troubleshoot container networking errors
+                </a>
+              </p>
+              <p>
+                <a href="https://docs.docker.com/docker-for-windows/troubleshoot/">
+                  Docker Desktop logs and troubleshooting
+                </a>
+              </p>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-xs-12 col-md-6">
+              <h2 class="title">Stack Overflow</h2>
+              <p>
+                Browse through some of the frequent questions asked about Docker
+                on Stack Overflow.
+              </p>
+              <p>
+                <a href="https://stackoverflow.com/questions/24319662/from-inside-of-a-docker-container-how-do-i-connect-to-the-localhost-of-the-mach">
+                  From inside of a Docker container, how do I connect to the localhost of the machine?
+                </a>
+              </p>
+              <p>
+                <a href="https://stackoverflow.com/questions/16047306/how-is-docker-different-from-a-virtual-machine">
+                  How is Docker different from a virtual machine?
+                </a>
+              </p>
+              <p>
+                <a href="https://stackoverflow.com/questions/21553353/what-is-the-difference-between-cmd-and-entrypoint-in-a-dockerfile">
+                  What is the difference between CMD and ENTRYPOINT in a Dockerfile?
+                </a>
+              </p>
+              <p>
+                <a href="https://stackoverflow.com/questions/22111060/what-is-the-difference-between-expose-and-publish-in-docker">
+                  What is the difference between “expose” and “publish” in Docker?
+                </a>
+              </p>
+              <p>
+                <a href="https://stackoverflow.com/questions/19234831/where-are-docker-images-stored-on-the-host-machine">
+                  Where are Docker images stored on the host machine?
+                </a>
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section id="community-resources" class="container">
+    <div class="row">
+      <div class="col-xs-12">
+        <h2 class="title">Community resources</h2>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-xs-12 col-sm-6 col-md-4 card-holder">
+        <a href="https://events.docker.com/chapters/" class="card">
+          <h5 class="title">Find a local meetup</h5>
+          <p>Attend one of the 200+ Docker Meetups around the globe.</p>
+        </a>
+      </div>
+      <div class="col-xs-12 col-sm-6 col-md-4 card-holder">
+        <a href="https://www.docker.com/events" class="card">
```

Comment:
Same for these links; even though they're on "our" domain; they're navigating away from the docs, so should probably open in a new tab/window